src/00_Introduction.md

```diff
@@ -19,3 +25,15 @@ builds upon mature technologies such as TypeScript, VS Code, Webpack, and Node.j
 
 This site provides all the details on how to use Gitpod and Theia. If you have questions
 or want discuss something, please join the [Gitpod community on Spectrum](https://spectrum.chat/gitpod).
+
+## Architecture
+
+Gitpod.io runs in multiple Kubernetes clusters hosted on Google Cloud infrastructure in 3 different regions:
```

Comment:
Nit: Maybe `three` instead of `3`?

```suggestion
Gitpod.io runs in multiple Kubernetes clusters hosted on Google Cloud infrastructure in three different regions:
```